repos.json

```diff
@@ -1,275 +1,410 @@
 [
     {
-        "compliant_cli": true,
-        "files": {
-            "Dockerfile": null,
-            "README.md": null,
-            "ocrd-tool.json": null,
-            "setup.py": null
-        },
-        "git": {
-            "last_commit": "Mon May 30 13:15:47 2022 +0200",
```

Comment:
Probably a good idea to use `YYYY-MM-DDTHH:mm:ssZ` for consistency.

Reply:
Since we omit this part, I don't see a reason for change here. But in case we re-introduce it, I'll keep this in mind!